config/initializers/assets.rb

```diff
@@ -6,7 +6,7 @@
 Rails.application.config.assets.enabled = true
 
 # Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.version = "1.2.5"
+Rails.application.config.assets.version = "2"
```

Comment:
Rails assets 2.0, here we go! 🚀 😄

Reply:
Lol, this value of this number doesn't actually mean anything, so I figured we don't need three segments ;)

Reply:
No asset version `1.3.0.beta1`? 😛